files/ceph-version-qemu.sh

```diff
@@ -30,11 +30,15 @@ PIDS=$(pidof /usr/libexec/qemu-kvm)
 
 for PID in $PIDS; do
   QEMU_TIMESTAMP=$(date -d "$(stat -c %x /proc/$PID/stat)" +%s)
-  if [[ "$QEMU_TIMESTAMP" -lt "$CEPH_UPGRADE_TIMESTAMP" ]]
+  INSTANCE_NAME=$(ps -ef -q $PID|grep -Po "(instance-\w+)"|uniq)
+  INSTANCE_QEMU_MON_RBD=$(virsh qemu-monitor-command --hmp "$INSTANCE_NAME" 'info block'|grep rbd)
```

Comment:
What does this do? INSTANCE_QEMU_MON_RBD

Reply:
Don't be afraid to comment your code.

Reply:
I got rid of my fears and commented